_posts/2022-04-20-WildFly-s2i-v2-Released.adoc

```diff
@@ -0,0 +1,232 @@
+
+=== Universal Base Image 8 Minimal
+
+Our new images are based on link:https://hub.docker.com/r/redhat/ubi8[Universal Base Image]. 
+The UBI8 image has been designed and engineered to be the base layer for containerized applications. 
+The Minimal flavor is a stripped down image.
+
+=== Relationship with the openjdk S2I images
+
+Our new images are sharing the Java VM configuration link:https://github.com/jboss-container-images/openjdk/blob/develop/modules/jvm/api/module.yaml[API] with the link:https://github.com/jboss-container-images/openjdk[openjdk images]. 
+This has multiple benefits:
+
+* JDK alignement. Benefit from latest updates.
+* JVM configuration alignment. Benefit from the same automatic tuning and configuration of Java VM.
+
+In the end you are getting an homogeneous user experience whatever the kind of WildFly server packaging you have chosen: 
+WildFly Bootable JAR that runs on the openjdk images or WildFly that runs on our new images.
+
+=== An S2I builder image un-coupled from WildFly server
+
+S2I builder images are generic and can be used to deploy application for any WildFly server version.
+
+Builder images have no dependency on a WildFly server release (as opposed to the legacy centos7 WildFly images that are bound to a given server version). 
+
+You can even run WildFly EE8 or WildFly EE9 based application on the same images.
+The link:https://github.com/wildfly/wildfly-s2i/tree/main/examples/postgresql/README.md[PostgreSQL example] and 
+link:https://github.com/wildfly/wildfly-s2i/tree/main/examples/postgresql-jakarta/README.md[PostgreSQL Jakarta EE9 example] 
+are identical applications deployed on different kind of WildFly server. Just a matter of choosing the right Galleon feature-packs to assemble the right server.
+
+=== Release of the new images
+
+Images release cadence is now un-related to the WildFly server release cadence. Images have their own lifecycle. New releases will be done to address fixes and new features.
+Released images are deployed in the link:http://quay.io/wildfly[quay.io/wildfly organization]
+For the one that love to be on the bleeding edge we are constantly deploying images under development to the link:http://quay.io/wildfly-snapshots[quay.io/wildfly-snapshots organization].
+
+=== Using the JDK 17 S2I builder image
+
+Applications are built and run similarly on JDK11 and JDK17. For JDK17 just make sure to have your maven 
+projects to use `3.3.2` version for the link:https://maven.apache.org/plugins/maven-war-plugin/[Maven war plugin]. 
+This link:https://github.com/wildfly/wildfly-s2i/tree/main/examples/postgresql-jakarta/README.md[example] shows how to configure the Maven war plugin version. 
+
+[NOTE]
+====
+When using the JDK17 image, you need to set the following env variable to workaround link:https://issues.redhat.com/browse/WFCORE-5835[WFCORE-5835]:
+```
+JAVA_OPTS_APPEND=--add-exports=jdk.naming.dns/com.sun.jndi.dns=ALL-UNNAMED
+```
+====
+
+=== Multi arch images preview
```

Comment:
I think this should be written as "Multiarch"

Reply:
Fixed